### DISCUSSION TITLE
docs/self-hosted/custom-theme

### DISCUSSION THREAD

**Discussion by giscus** [flowershow/flowershow#599] [comments]
# docs/self-hosted/custom-theme

Learn how to add custom fonts and colors to your site

https://flowershow.app/docs/self-hosted/custom-theme

**Comment by ghost** [comment]
Hi!

(1) How do I change the colour of h1, 2, 3 headings etc., to be different from the colour of the main body text?

(2) How do I change the text size of headings and main body text?

(3) How do I change the colour of the text in the LHS TOC?

Thank you in advance!
Chris

**Comment by lcatalanm** [comment]
All the images in your website are down.

**Comment by olayway** [comment]
Closing this issue since it concerns the old, now-deprecated self-hosted Flowershow Next.js template.
